doc/connecting-lab.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Connecting LAVA Lab to the pipeline instance"
+date: 2024-02-19
+description: "Connecting a LAVA lab to the KernelCI pipeline"
+weight: 2
+---
+
+As we are moving towards the new KernelCI API and pipeline, we need to make sure
+all the existing LAVA labs are connected to the new pipeline instance.  This
+document explains how to do this.
+
+## Token setup
+
+The first step is to generate a token for the lab.  This is done by the lab admin,
+and the token is used to submit jobs from pipeline to the lab, and to authenticate
+LAVA lab callbacks to the pipeline.
+Requirements for the token:
+- Description: a string matching the regular expression `[a-zA-Z0-9\-]+`, for example "kernelci-new-api-callback"
+- Value: arbitrary, kept secret
+
+## Pipeline configuration
+
+### Secrets (toml) file
+
+The next step is to add the token to the pipeline services configuration files.
+Secrets are stored in the `kernelci.secrets` section of the `kernelci.toml` file,
+and added manually by the KernelCI system administrators.  For example, the token
+can be added to the `runtime.lava-labname` section of the `services.toml` file:
+
+```toml
+[runtime.lava-labname]
+runtime_token="TOKEN-VALUE"
+```
+
+### docker-compose file
+
+You need to add lab name to the scheduler-lava service in the `docker-compose.yml` file.
+
+### yaml pull request
+
+The final step is to submit a pull request to the `kernelci-pipeline` repository
```

Comment:
I guess we should also mention here that the pipeline uses `config/pipeline.yaml` (and a link to the file) for configurations before describing about sending in a PR.